website/content/docs/oss/installing/dev-mode.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+page_title: What is Dev Mode?
+description: Getting started with Boundary OSS in Dev mode
+---
+
+# What is Dev Mode?
+
+Before getting started with Boundary OSS, it's important to understand a few key
+concepts. Please consult our [Boundary concepts](/docs/concepts) page to
+familiarize yourself with Boundary's architecture, and terminology.
+
+Dev mode is an all-in-one installation method for getting started with Boundary
+quickly. As the name implies, dev mode is not a production installation method,
+but provides a way to get Boundary running with a Postgres database easily. Do
```

Comment:
It might be worth calling out that it doesn't get you started with your _own_ Postgres database easily; it only launches one in a Docker container and (by default) destroys the container afterwards. (It is possible to point it to a different database, but this is sort of advanced/debug usage only.)

Reply:
this seems covered by the statement that follows the components:

```
Dev mode brings up a fully-functioning instance of Boundary, including:

- A controller server
- A worker server
- A Postgres database

These components should all be considered ephemeral - no data persistence occurs
across dev mode restarts.
```

Reply:
Sure, I guess I just don't see the reason why "with a Postgres database..." needs to be a part of the sentence. Simply saying "...but provides a way to get Boundary running easily" seems to cover it.

Reply:
I don't have any attachment to whether it's included or not, but at a glance this seems useful since you see the postgres container running under docker